src/features/VisualNovel/pages/MediaPage/MediaPage.tsx

```diff
@@ -46,7 +42,7 @@ export const MediaPage: VFC = () => {
     <Flex gridGap="5" flexWrap="wrap">
       {filteredScreens.map((screen, idx) => (
         <Image
-          onClick={() => handleClickOnImage(idx)}
+          onClick={() => handleImageClick(idx)}
```

Comment:
Accordingly to the comment above.
```suggestion
          onClick={handleImageClick(idx)}
```